org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/ast/Expression.java

```diff
@@ -68,7 +69,7 @@
 public abstract class Expression extends Statement {
 
 	public Constant constant;
-
+	public StringBuilder concatenatedLiteral;
```

Comment:
Could this already be the String that is to be made from the StringBuilder? I cannot really spot code that would append to the StringBuilder after it was set as a field here.

Reply:
I am getting rid of this field altogether as @srikanth-sankaran suggested.